src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -153,6 +160,39 @@ export default class UniversalDiscoveryModule extends Component {
         }));
     }
 
+    dispatchBookmarkChangeEvent(locationId, bookmarked) {
+        const event = new CustomEvent('ez-bookmark-change', {
+            detail: {
+                bookmarked,
+                locationId,
+            },
+        });
+
+        document.body.dispatchEvent(event);
+    }
+
+    onBookmarkRemoved(item) {
+        const { activeTab } = this.state;
+
+        if (this._refBookmarksPanelComponent) {
+            this._refBookmarksPanelComponent.onBookmarkRemoved(item);
```

Comment:
You should use either state or props to manipulate the behavior of compound components.